fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/FixityResultImpl.java

```diff
@@ -91,16 +92,19 @@ public String getStoreIdentifier() {
 
     @Override
     public boolean equals(final Object obj) {
-
-        boolean result = false;
         if (obj instanceof FixityResult) {
             final FixityResult that = (FixityResult) obj;
-            result =
-                    computedSize == that.getComputedSize() &&
-                            computedChecksum.equals(that.getComputedChecksum());
+            return computedSize == that.getComputedSize() &&
```

Comment:
Please add a minor update the the following unit test:
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/utils/FixityResultImplTest.java#L38